src/main/java/btree4j/utils/lang/ArrayUtils.java

```diff
@@ -300,21 +301,29 @@ public static int lastIndex(final Object array) {
      * @throws IndexOutOfBoundsException if the index is out of range (index < 0 || index >=
      *         array.length), or if the array is <code>null</code>.
      * @since 2.1
+     * @implNote Its dependance on removeFrom may slow down the function.
```

Comment:
Existing function `remove(final T[] array, final int index)` should be as it is.

Reply:
Fixed.